lib/fog/ovirt/compute/v4.rb

```diff
@@ -88,6 +88,7 @@ def get_attr_value(value, opts)
               opts[:ha] = value.enabled
               value
             when OvirtSDK4::TemplateVersion
+              require "ostruct"
```

Comment:
I think we can move it to the head of the file.
We are not loading too much here that it's worth hiding this `require` in the code.

Reply:
I was trying to follow the existing coding style. This pattern is used throughout the file.

Reply:
It's a bit different case: there are `require "ovirtsdk4"` statements in code, because we had support for both v3 and v4, and you had to select which one are you using.
In the current case, it's something that is related to the data structure that is returned, so it is more similar to the `require "active_support"` statement that we have in the header

Reply:
Thanks for clarifying this. Updated the PR. 

Would really appreciate it if you could have a look at my other PRs as well.

Reply:
Just tag me on the PR, and hopefully I will get to it